src/ui/components/Library/Navigation/TeamButton.tsx

```diff
@@ -43,25 +45,73 @@ export function TeamButton({
   };
 
   return (
-    <Link
-      href={url}
+    <div
       className={classNames(
-        `${styles.teamRow} group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
-        isSelected ? `${styles.teamRowActive} cursor-auto` : "cursor-pointer"
+        `flex flex-col ${styles.teamRow} `,
+        isSelected ? `${styles.teamRowActive}` : ""
       )}
-      onClick={onClick}
     >
-      <span className="overflow-hidden overflow-ellipsis whitespace-pre">
-        <div className="flex">
-          <LibraryIcon teamType={isTest ? "tests" : "team"} />
-          {label}
+      <Link
+        href={url}
+        className={classNames(
+          `group flex flex-row justify-between space-x-2 px-4 py-2 text-left transition duration-200 hover:text-white focus:outline-none`,
+          isSelected ? `cursor-auto` : "cursor-pointer"
+        )}
+        onClick={onClick}
+      >
+        <span className="overflow-hidden overflow-ellipsis whitespace-pre">
+          <div className="flex">
+            <LibraryIcon teamType={isTest ? "tests" : "team"} />
+            {label}
+          </div>
+        </span>
+        {isNew ? (
+          <div className={"rounded-md bg-primaryAccent px-3 py-0.5 text-xs text-white"}>New</div>
+        ) : null}
+        {showSettingsButton ? <SettingsButton /> : null}
+      </Link>
+      {isSelected ? <TestTeamViews /> : null}
+    </div>
+  );
+}
+
+// We don't currently have a context shared between the navigation
+// and the content of the library UI. Getting the view/teamId this
+// way is pretty clunky, so we should probably lift that state up
+function TestTeamViews() {
+  const [showTestsView] = useLocalStorageUserData("enableTestView");
+  const router = useRouter();
+  const cont = useContext(ViewContext);
+  const view = useGetTeamRouteParams().view;
+  const { teamId } = useGetTeamRouteParams();
+
+  const setView = (view: View) => {
+    pushRoute(router, `/team/${teamId}/${view}`);
+  };
+
+  return (
+    <div className="pl-4 pb-2">
+      <div className="flex flex-col">
+        <div
+          className={`px-4 py-1 ${
```

Comment:
The margin and alignment for this sub menu looks awkward to me. The "Runs" text is not indented enough to look "nested" beneath the team name:

<img width="266" alt="Screen Shot 2023-11-15 at 10 20 58 AM" src="https://github.com/replayio/devtools/assets/29597/86abb840-f294-4cea-bb6a-2951490daf83">

And it looks lopsided vertically because of the added bottom padding:

<img width="269" alt="Screen Shot 2023-11-15 at 10 21 12 AM" src="https://github.com/replayio/devtools/assets/29597/cc2bee3f-9942-4577-864a-4169510ae351">

I suggest changing it like so for now:
```suggestion
          className={`px-8 ${
```

<img width="260" alt="Screen Shot 2023-11-15 at 10 22 55 AM" src="https://github.com/replayio/devtools/assets/29597/752a8f98-8bb7-4c98-98f0-23d6bf4e6f51">